website/docs/reference/node-selection/syntax.md

```diff
@@ -205,7 +205,7 @@ The available options depend on the resource (node) type:
 The state and result selectors can also be combined in a single invocation of dbt to capture errors from a previous run OR any new or modified models.
 
 ```bash
-dbt run --select "result:<status>+ state:modified+ --defer --state ./<dbt-artifact-path>"
+dbt run --select result:<status>+ state:modified+ --defer --state ./<dbt-artifact-path>
```

Comment:
Add quotes for the `--select` arguments per [here](https://docs.getdbt.com/reference/node-selection/syntax#specifying-resources) (and https://github.com/dbt-labs/docs.getdbt.com/issues/4040):

```suggestion
dbt run --select "result:<status>+" state:modified+ --defer --state ./<dbt-artifact-path>
```